src/components/__stories__/modal.tsx

```diff
@@ -0,0 +1,52 @@
+import { action, storiesOf } from "@kadira/storybook"
+import * as React from "react"
+
+import Button from "../buttons/default"
+import Modal from "../modal/modal"
+import ModalHeader from "../modal_header"
+import Title from "../title"
+
+const ModalStyle = {
```

Comment:
we should have default width and height in the Modal component itself so the parent doesn't have to pass in styling